first german translation

ingorichter opened the pull request:
Don't merge yet.

ssorallen commented:
This is fantastic, thank you. I'm looking forward to merging and shipping our first translation.

ssorallen commented:
I extracted a few more strings in https://github.com/tabwrangler/tabwrangler/commit/626db7c21813e99d81cd9350fb4498ae438e21cf for translation. That commit should merge into your PR cleanly if you want to look at those strings too.

ssorallen commented:
Do you think it would be worthwhile to merge the translations you completed and leave the rest in English, or is that too strange? A half-English half-German UI.

ingorichter commented:
Let me finish this tonight. I'll update my PR tonight and let you know.
Thanks for the reminder.
On Tue, Feb 27, 2018 at 21:18 Ross Allen <notifications@github.com> wrote:

> Do you think it would be worthwhile to merge the translations you
> completed and leave the rest in English, or is that too strange? A
> half-English half-German UI.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tabwrangler/tabwrangler/pull/183#issuecomment-369124676>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFQxmi1a8ugfMPggZj905cyyjYlFYzLks5tZOGVgaJpZM4RFE67>
> .
>
-- 


[image: --]
Ingo Richter
[image: https://]about.me/ingorichter
<https://about.me/ingorichter?promo=email_sig>

ingorichter commented:
@ssorallen I've merged master and added the missing loc strings. This should be ready to go after this https://github.com/tabwrangler/tabwrangler/pull/209 landed and I added the new loc string to the German translation

ssorallen commented:
`"corral_tabSessionFresh"` is still in English. Did you want to translate that one too? It's a tooltip for the new session restoring ability. If you hover over the leaf icon on the right of recently closed tabs, you'll see the string, like in this screenshot:

<img width="679" alt="screen shot 2018-03-05 at 5 44 09 pm" src="https://user-images.githubusercontent.com/29612/36968219-e0ec9320-209c-11e8-9a78-faf32512bc5b.png">

ingorichter commented:
Thanks. I'll translate it too

ingorichter commented:
I've updated the PR with the missing translation and a couple of minor loc improvements

ssorallen commented:
Thank you for another great contribution, @ingorichter! This will be in the next release.
